README.md

```diff
@@ -33,6 +33,6 @@ Before you get ahead of yourself, though, please read our
 
 ## License
 
-[MIT - Hippocratic](https://firstdonoharm.dev)
+[Hippocratic 2.1](https://firstdonoharm.dev)
 
-Copyright (c) 2015-present, Humanity Codes LLC
+Copyright (c) 2020-Present, Humanity Codes LLC
```

Comment:
Do you mind updating the year to be `2015` here? That's when the slackbot was originally created. 🙂

Reply:
Sorry, Copy and paste error.